### PR TITLE
Update json and md file as per code

### PR DESCRIPTION
**Pull request by Caghera**
Updated the DisplaySettings.json and DisplaySettings.md as per the code

Jira:- https://ccp.sys.comcast.net/browse/RDK-34103

### CODE REVIEW

**Comment by anand-ky** [comment]
@apatel859, @ddevad,
Pls review these documentation changes for DisplaySettings APIs.

**Review by apatel859 — APPROVED** [review]

File: DisplaySettings/DisplaySettings.json [hunk 1]
```diff
@@ -1166,8 +1208,8 @@
                 ]
             }
         },
-        "getSurroundVirtualizer":{
-            "summary": "(Version 2) Returns the current surround virtualizer boost settings",
+        "getSurroundVirtualizer(v2)":{
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:1211]: Remove `(v2)` from the method signature. Application calls that include `(v2)` in the method signature will fail.

File: DisplaySettings/DisplaySettings.json [hunk 2]
```diff
@@ -1342,8 +1404,8 @@
                 ]
             }
         },
-        "getVolumeLeveller": {
-            "summary": "(Version 2) Returns the current Volume Leveller setting",
+        "getVolumeLeveller(v2)": {
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:1407]: Remove (v2) from the method signature. Application calls that include (v2) in the method signature will fail.

File: DisplaySettings/DisplaySettings.json [hunk 3]
```diff
@@ -1869,8 +2045,8 @@
                 "$ref": "#/definitions/result"
             }
         },
-        "setSurroundVirtualizer":{
-            "summary": "(Version 2) Sets the Surround Virtualizer boost. The Speaker/Surround Virtualizer enables a surround sound signal (including one generated by the Surround Decoder) to be rendered over a device with built-in speakers or headphones.",
+        "setSurroundVirtualizer(v2)":{
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:2048]: Remove `(v2)` from the method signature. Application calls that include `(v2)` in the method signature will fail.

File: DisplaySettings/DisplaySettings.json [hunk 4]
```diff
@@ -1970,8 +2145,8 @@
                 "$ref": "#/definitions/result"
             }
         },
-        "setVolumeLeveller":{
-            "summary": "(Version 2) Sets the Volume Leveller level. Volume Leveler is an advanced volume-control solution that maintains consistent playback levels for content from different sources.",
+        "setVolumeLeveller(v2)":{
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:2148]: Remove `(v2)` from the method signature. Application calls that include `(v2)` in the method signature will fail.

File: DisplaySettings/DisplaySettings.json [hunk 5]
```diff
@@ -1132,7 +1174,7 @@
             }
         },
         "getSurroundVirtualizer":{
-            "summary": "Returns the current surround virtualizer boost settings",
+            "summary": "(Version 1) Returns the current surround virtualizer boost settings.\n \n### Event \n\n No Events.",
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:1177]: The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

File: DisplaySettings/DisplaySettings.json [hunk 6]
```diff
@@ -1308,7 +1370,7 @@
             }
         },
         "getVolumeLeveller": {
-            "summary": "Returns the current Volume Leveller setting",
+            "summary": "(Version 1)Returns the current Volume Leveller setting.\n \n### Event \n\n No Events.",
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:1373]: The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

File: DisplaySettings/DisplaySettings.json [hunk 7]
```diff
@@ -1841,16 +2018,15 @@
...
+                    "soundMode"
                 ]
             },
             "result": {
                 "$ref": "#/definitions/result"
             }
         },
         "setSurroundVirtualizer":{
-            "summary": "Sets the Surround Virtualizer boost. The Speaker/Surround Virtualizer enables a surround sound signal (including one generated by the Surround Decoder) to be rendered over a device with built-in speakers or headphones.",
+            "summary": "(Version 1)Sets the Surround Virtualizer boost. The Speaker/Surround Virtualizer enables a surround sound signal (including one generated by the Surround Decoder) to be rendered over a device with built-in speakers or headphones.\n \n### Event \n\n No Events.",
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:2029]: The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

File: DisplaySettings/DisplaySettings.json [hunk 8]
```diff
@@ -1951,7 +2126,7 @@
             }
         },
         "setVolumeLeveller":{
-            "summary": "Sets the Volume Leveller level. Volume Leveler is an advanced volume-control solution that maintains consistent playback levels for content from different sources.",
+            "summary": "(Version 1)Sets the Volume Leveller level. Volume Leveller is an advanced volume-control solution that maintains consistent playback levels for content from different sources.\n \n### Event \n\n No Events.",
```
Comment (jruzzi) [inline DisplaySettings/DisplaySettings.json:2129]: The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

**Review by jruzzi — CHANGES_REQUESTED** [review]
We cannot change the method signature. The method name must remain exactly how an application makes the call. Hopefully, once the Thunder tooling supports versioning we will be able to remove all manual versioning conventions. After making the changes update the markdown accordingly.

File: DisplaySettings/DisplaySettings.json (hunk 1 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json]: As discussed keeping only version2 methods and removed (v2) from method.

File: DisplaySettings/DisplaySettings.json (hunk 2 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json]: As discussed keeping only version2 methods and removed (v2) from method.

File: DisplaySettings/DisplaySettings.json (hunk 3 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json]: As discussed keeping only version2 methods and removed (v2) from method.

File: DisplaySettings/DisplaySettings.json (hunk 4 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json]: As discussed keeping only version2 methods and removed (v2) from method.

File: DisplaySettings/DisplaySettings.json (hunk 5 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json:1177]: As discussed keeping only version2 methods in md file.

File: DisplaySettings/DisplaySettings.json (hunk 8 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json:2129]: As discussed keeping only version2 methods in md file.

File: DisplaySettings/DisplaySettings.json (hunk 7 above)
Comment (Caghera) [reply DisplaySettings/DisplaySettings.json:2029]: As discussed keeping both versions in JSON file and only version2 methods in md file.

**Review by jruzzi — APPROVED** [review]

**Pull request merged**